terraform/email/main.tf

```diff
@@ -49,7 +59,16 @@ resource "aws_route53_record" "primary_amazonses_dmarc_record" {
   name    = "_dmarc.${var.domain}"
   type    = "TXT"
   ttl     = "60"
-  records = ["v=DMARC1; p=none; rua=${var.dmarc}; fo=1; adkim=s; aspf=s"]
+  records = ["v=DMARC1; p=none; rua=${var.dmarc}; fo=1; adkim=r; aspf=r"]
+}
+
+resource "aws_route53_record" "primary_amazonses_mx_record" {
+  provider = "aws.email"
```

Comment:
This doesn't need to be set, Route 53 is a global service, although I don't think it hurts since again, Route 53 is a global service, so I think it's fine, just noting!

Reply:
Ah, I think you're right. I set this when I was trying to figure out how to get the right region for the templated record below. Resolved with 1ed6ee6